cli/cliui/output.go

```diff
@@ -83,6 +83,10 @@ func (f *OutputFormatter) Format(ctx context.Context, data any) (string, error)
 	return "", xerrors.Errorf("unknown output format %q", f.formatID)
 }
 
+func (f *OutputFormatter) FormatID() string {
+	return f.formatID
+}
```

Comment:
Took me a second to realize this `formatID` is updated here:

https://github.com/coder/coder/blob/a12f553130a56ec2add6f7b7786b38a642844770/cli/cliui/output.go#L64-L70

Just throw a comment to make the behavior clear.

```suggestion
// FormatID will return the ID of the format selected by `--output`.
// If no flag is present, it returns the 'default' formatter.
func (f *OutputFormatter) FormatID() string {
	return f.formatID
}
```

Reply:
I added a comment.